src/summary/TabularListVisualizer.jsx

```diff
@@ -297,43 +299,64 @@ class TabularListVisualizer extends Component {
-                        <td width={colSize}
-                            key={elementId}
-                        >
-                            <span>
-                                {elementText}
-                            </span>
-                        </td>
-                    );
-                } else {
-                    columnItem = (
-                        <td width={colSize}
-                            className={itemClass} 
-                            data-test-summary-item={item[0].value} 
-                            key={elementId}
-                        >
-                            <span>
-                                {elementText}
-                            </span>
-                        </td>
-                    );
                 }
+                else if (this.props.allowItemClick && isInsertable) {
+                        // Get value off of element given two cases:
+                        // 1. Element type is shortcut, value is returned by element.value()
+                        // 2. Element type is string, the value is just the string
+
+                        // Make unique id for each value
+                        columnItem = (
+                            <td width={colSize}
+                                className={itemClass}
+                                key={elementId}
+                            >
+                            {this.renderedStructuredData(item[0].value, element, elementId, elementText)}
+                            </td>
+
+                        );
+                    } else if (!isInsertable) {
+                        columnItem = (
+                            <td width={colSize}
+                                key={elementId}
+                            >
+                                <span>
+                                    {elementText}
+                                </span>
+                            </td>
+                        );
+                    } else {
+                        columnItem = (
+                            <td width={colSize}
+                                className={itemClass}
+                                data-test-summary-item={item[0].value}
+                                key={elementId}
+                            >
+                                <span>
+                                    {elementText}
+                                </span>
+                            </td>
+                        );
+                    }
+
+                    if(!Lang.isNull(elementText) && elementText.length > 100){
+                        const text = <span>{longElementText}</span>
+                        columnItem = (
+                            <Tooltip
+                                key={elementId}
+                                overlayStyle={{'visibility': true}}
+                                placement="top"
+                                overlayClassName={`context-panel-tooltip large`}
```

Comment:
[DL1] Should we use a different overlayClassName here since this Tooltip is in the targeted data panel?